org.eclipse.scout.rt.mom.jms/src/main/java/org/eclipse/scout/rt/mom/jms/JmsMomImplementor.java

```diff
@@ -291,7 +296,9 @@ protected JmsSessionProviderWrapper createSessionProviderWrapper(boolean transac
 
   protected JmsSubscription createJmsSubscription(IDestination<?> destination, SubscribeInput input, IJmsSessionProvider sessionProvider, IFuture<?> worker) {
     JmsSubscription subscription = new JmsSubscription(destination, input, sessionProvider, worker);
-    subscription.awaitStarted(m_subscriptionAwaitStartedSeconds, TimeUnit.SECONDS);
+    subscription.awaitStarted(
+        m_subscriptionAwaitStartedSeconds != -1 ? m_subscriptionAwaitStartedSeconds : null,
```

Comment:
Here -1 means 'infinite', in JmsSubscription instead a value of null indicated 'infinte'.
Suggestion: 
- Add constant public static int JmsMomImplementor.WAIT_TIME_INFINITE = -1;
- use constant as default value in JmsMomImplementor if no (or a null) config value is set in properties
- pass value as int to JmsSubscription
- change subscription to check for waitTime != JmsMomImplementor.WAIT_TIME_INFINITE to consider if the Thread should be blocked waiting for the subscription